### PR TITLE
Refact in UpdateAssemblyInfo

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/UpdateAssemblyInfo/Program.cs [hunk 1]
```diff
@@ -103,7 +103,7 @@ private static int Main(string[] args)
         private static string GetGitPath()
         {
             var envPath = Environment.GetEnvironmentVariable("PATH");
-            if (!string.IsNullOrWhiteSpace(envPath))
+            if (!string.IsNullOrWhiteSpace(envPath) && envPath.IndexOf("git.exe", StringComparison.Ordinal) > 0)
```
Comment (xylographe) [inline src/UpdateAssemblyInfo/Program.cs]: PATH contains directory paths separated by `';'` (Windows)  or `':'` (Posix).

Comment (ivandrofly) [inline src/UpdateAssemblyInfo/Program.cs]: yeah but instead of looping looking for git.ext why not checking in path first

Comment (xylographe) [inline src/UpdateAssemblyInfo/Program.cs]: Like I said, PATH contains *directory* paths. PATH does not contain *file* paths.

Comment (ivandrofly) [inline src/UpdateAssemblyInfo/Program.cs]: Ah... okay :+1:

File: src/UpdateAssemblyInfo/Program.cs [hunk 2]
```diff
@@ -113,7 +112,7 @@ private static string GetGitPath()
                 }
             }
 
-            var gitPath = Path.Combine("Git", "bin", "git.exe");
+            var gitPath = "Git\\bin\\git.exe";
```
Comment (niksedk) [inline src/UpdateAssemblyInfo/Program.cs]: I think it's just nicer to always use the platform independent Path.Combine method

**Pull request closed without merging**